person.rb

```diff
@@ -0,0 +1,25 @@
+# person.rb
+
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(name = 'Unknown', age = nil, parent_permission: true)
+    @id = generate_id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age.to_i >= 18
+  end
+
+  def generate_id
+    "P#{rand(1000..9999)}"
+  end
+
+  def can_use_services?
+    true
+  end
```

Comment:
- [ ] The requirement was to create a Public method `can_use_services?` that returns true if `person` is of age or if they have permission from parents. your code returns true but does not state the condition for this.
- Kindly write the logic 👍 
### Requirement:
![image](https://github.com/jlcarrascof/OOP-school-library/assets/50990925/256d4030-1986-42dd-a2ef-364c7698995d)

Reply:
I fixed the logic, in this topic, Mr. Sunday to make can_use_services? as a public method.

![fix-01](https://github.com/jlcarrascof/OOP-school-library/assets/86327867/d7cd3268-c71f-4aaa-aba2-cbacf6b1d20c)